src/_data/sidenav.yml

```diff
@@ -559,6 +559,9 @@
           permalink: /tools/devtools/inspector
         - title: Performance view
           permalink: /tools/devtools/performance
+          children:
+          - title: Performance profiling for web
+            permalink: /tools/devtools/web-performance
```

Comment:
Since this is outside of DevTools, what do you think about placing this doc under "Performance & optimization" next to "Performance profiling": https://github.com/flutter/website/blob/1b8b776f245743ea5965544e4eae5bad75faae15/src/_data/sidenav.yml#L472-L473

Reply:
Sure! Moved it there